### PR TITLE
Remove cell hover events for mobile devices

### PR DESCRIPTION
**Pull request by broskoski**
Removes weird click behavior for mobile. I welcome alternative approaches.

### CODE REVIEW

File: src/v2/util/isMobile.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export function isMobile() {
+  return (
+    typeof window.orientation !== 'undefined' ||
+    navigator.userAgent.indexOf('IEMobile') !== -1
```
Comment (dzucconi) [inline src/v2/util/isMobile.ts]: Shouldn't you be detecting touch?

Comment (broskoski) [reply src/v2/util/isMobile.ts]: Apparently the presence of  `window.orientation` gets the most coverage for mobile devices. the `IEMobile` is just to catch that Windows mobile devices.

Comment (dzucconi) [reply src/v2/util/isMobile.ts]: I don't get it though — this is explicitly because of touch devices. It has nothing to do with mobility.

Comment (dzucconi) [reply src/v2/util/isMobile.ts]: What about touch desktops?

Comment (broskoski) [reply src/v2/util/isMobile.ts]: Ok, good point

File: src/v2/util/isTouchDevice.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// Extracted from https://stackoverflow.com/posts/4819886/revisions
+export function isTouchDevice() {
+  return 'ontouchstart' in window || navigator.maxTouchPoints
+}
```
Comment (dzucconi) [inline src/v2/util/isTouchDevice.ts]: https://github.com/aredotna/ervell/blob/master/src/v2/util/is.js#L13

Comment (broskoski) [reply src/v2/util/isTouchDevice.ts]: Thanks, will remove.

**Pull request merged**